plugin/trino-hive/src/main/java/io/trino/plugin/hive/security/AccessControlMetadata.java

```diff
@@ -108,20 +108,36 @@ default void revokeRoles(ConnectorSession connectorSession, Set<String> roles, S
         throw new TrinoException(NOT_SUPPORTED, "This connector does not support roles");
     }
 
+    /**
+     * Grants the specified privilege to the specified user on the specified schema
+     */
+    default void grantSchemaPrivileges(ConnectorSession session, String schemaName, Set<Privilege> privileges, HivePrincipal grantee, boolean grantOption)
+    {
+        throw new TrinoException(NOT_SUPPORTED, "This connector does not support grants on schemas");
+    }
+
+    /**
+     * Revokes the specified privilege on the specified schema from the specified user
+     */
+    default void revokeSchemaPrivileges(ConnectorSession session, String schemaName, Set<Privilege> privileges, HivePrincipal grantee, boolean grantOption)
+    {
+        throw new TrinoException(NOT_SUPPORTED, "This connector does not support revokes on schemas");
+    }
+
     /**
      * Grants the specified privilege to the specified user on the specified table
      */
     default void grantTablePrivileges(ConnectorSession session, SchemaTableName tableName, Set<Privilege> privileges, HivePrincipal grantee, boolean grantOption)
     {
-        throw new TrinoException(NOT_SUPPORTED, "This connector does not support grants");
+        throw new TrinoException(NOT_SUPPORTED, "This connector does not support grants on tables");
```

Comment:
nit: I guess this can be captured in a different commit.

Reply:
I think this is fine, because without this commit the change is not needed. Doing it sooner or later would be awkward to me.